Dockerfile

```diff
@@ -12,6 +12,10 @@ ENV LINK_SCHEME redis
 ENV LINK_PASSWORD password
 ENV LINK_PATH /0
 
+RUN mkdir /data && chown nobody:nobody /data
```

Comment:
`nobody` is the user running the command inside the container: https://github.com/convox/redis/blob/master/docker-entrypoint.sh#L5